tasks/section5.yml

```diff
@@ -1,5 +1,5 @@
 - name: "SCORED | 5.1.1 | AUDIT | Ensure cron daemon is enabled"
-  command: /bin/true
+  shell: systemctl is-enabled crond | grep enabled
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*